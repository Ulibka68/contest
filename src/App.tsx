import React from "react";
import { Chart } from "@/modules/Chart";
import { Provider } from "react-redux";
import { store } from "@/store";
export const App: React.FC = () => (
  <Provider store={store}>
    <Chart />
  </Provider>
);
