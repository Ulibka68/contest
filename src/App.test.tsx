import React from "react";
import { shallow } from "enzyme";
import { App } from "@/App";

describe("Integration test full app", () => {
  it("to match snapshot", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.render()).toMatchSnapshot();
  });
});
