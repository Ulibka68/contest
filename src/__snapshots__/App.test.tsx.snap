// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Integration test full app to match snapshot 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {
          "style": "text-align:right",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "css-19bgl6d",
              "style": "border-color:rgb(16, 139, 227);background-color:rgb(16, 139, 227)",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "style": "color:rgb(16, 139, 227)",
                },
                "children": Array [
                  Object {
                    "data": "Night mode",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "span",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "style": undefined,
                },
                "x-attribsPrefix": Object {
                  "style": undefined,
                },
              },
            ],
            "name": "button",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
              "style": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
              "style": undefined,
            },
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "style": undefined,
        },
        "x-attribsPrefix": Object {
          "style": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
